src/main/java/org/tron/p2p/connection/business/handshake/HandshakeService.java

```diff
@@ -45,10 +52,12 @@ public void processMessage(Channel channel, Message message) {
     if (channel.isActive()) {
       if (msg.getCode() != DisconnectCode.NORMAL.getValue()
           || (msg.getNetworkId() != networkId && msg.getVersion() != networkId)) {
+        DisconnectCode disconnectCode = DisconnectCode.forNumber(msg.getCode());
```

Comment:
The other party has already sent a disconnect message. There is no need to send it again here.

Reply:
This only print the reason, doesn't send a disconnect message.